src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1856,25 +1884,34 @@ private Matches() {}
     });
   }
 
-  public static final Match<Territory> territoryIsBlockadeZone = Match.of(t -> {
-    final TerritoryAttachment ta = TerritoryAttachment.get(t);
-    if (ta != null) {
-      return ta.getBlockadeZone();
-    }
-    return false;
-  });
-
-  public static final Match<UnitType> UnitTypeIsConstruction = Match.of(type -> {
-    final UnitAttachment ua = UnitAttachment.get(type);
-    if (ua == null) {
+  static Match<Territory> territoryIsBlockadeZone() {
+    return Match.of(t -> {
+      final TerritoryAttachment ta = TerritoryAttachment.get(t);
+      if (ta != null) {
+        return ta.getBlockadeZone();
+      }
       return false;
-    }
-    return ua.getIsConstruction();
-  });
+    });
+  }
+
+  /**
+   * Returns a match indicating the specified unit type is a construction unit type.
+   */
+  public static Match<UnitType> unitTypeIsConstruction() {
+    return Match.of(type -> {
+      final UnitAttachment ua = UnitAttachment.get(type);
+      if (ua == null) {
+        return false;
+      }
+      return ua.getIsConstruction();
```

Comment:
This return statement could be merged with the if statement: `return ua != null && ua.getIsConstruction();`